composer/callbacks/checkpoint_saver.py

```diff
@@ -432,7 +488,13 @@ def _save_checkpoint(self, state: State, logger: Logger):
         )
         log.debug(f'Checkpoint locally saved to {saved_path}')
 
+        self.symlink_count += 1
```

Comment:
because now we upload symlink much later after the checkpoint files uploading finish. So there might be multiple local symlink files in the same tmp directory. I need to name them differently, so use the symlink_count to do that.